flutter_example/lib/services/ab-testing-services/ab_testing_service.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:async';
+
+import 'package:flutter_example/models/user.dart';
+import 'package:growthbook_sdk_flutter/growthbook_sdk_flutter.dart';
+
+import 'network_client.dart';
+
+/// A singleton service class wrapper around the [GrowthBookSDK].
+class ABTestService {
+  const ABTestService._();
+  static const ABTestService _instance = ABTestService._();
+  static ABTestService get instance => _instance;
+
+  static GrowthBookSDK? _sdkInstance;
+
+  /// Initializes the [GrowthBookSDK] if not initialized.
+  Future<void> initialize() async {
+    _sdkInstance ??= await GBSDKBuilderApp(
+      client: const ABtestingServiceNetworkClient(),
+      apiKey: '<API_KEY>',
+      hostURL: '<HOST_URL>',
```

Comment:
The goal of the examples repository is to provide working examples. This will not work. In the other projects, there are SDK endpoint URL's you can use. The data that they touch will not change (e.g. the Acme Donut company stuff). 

Here's an example URL: https://cdn.growthbook.io/api/features/java_NsrWldWd5bxQJZftGsWKl7R2yD2LtAK8C8EUYh9L8

In the JVM Java Spring project there's also an example for encrypted features if you'd like to provide that but as long as one is working, that would be great.

Reply:
If you don't want to use the example Acme Donuts one, you can create a project with your own values (I see you've got an example for "android app update." Feel free to provide an example of that.

Reply:
- Hey, @tinahollygb this totally makes sense to me. I would like to continue with the android update example. 
-  So, is there any organization's account for examples where I can create a separate project and use that for this example?

Reply:
You can create a new account and populate the features as you need. If you don't need to add anything else, you can use the existing ones. Alternatively you can implement multiple feature repositories, up to you. You can also do what the next.js example does, which gives detailed instructions on which features to set up and where to put API keys. That, I imagine, might be more work than ideal in Flutter, so it might make sense to make your own and then I can test it out.